x-pack/legacy/plugins/ml/public/data_frame/pages/data_frame_new_pivot/components/step_define/step_define_form.tsx

```diff
@@ -389,6 +444,21 @@ export const StepDefineForm: SFC<Props> = React.memo(({ overrides = {}, onChange
     </Fragment>
   );
 
+  const sourceDocsUrl = `https://www.elastic.co/guide/en/elasticsearch/reference/${metadata.branch}/query-dsl.html`;
+  const advancedSourceEditorHelpText = (
+    <Fragment>
+      {i18n.translate('xpack.ml.dataframe.stepDefineForm.advancedSourceEditorHelpText', {
+        defaultMessage:
+          'The advanced editor allows you to edit the source configuration of the data frame transform.',
```

Comment:
I think `source query clause` might be more accurate here, as you are only editing this part of the source configuration object (and not the index for example).

Reply:
Updated in 261ed3bc5b2a487226934e5f31aa2f5c9c3ebf1f